packages/kb-dashboard-tools/src/kb_dashboard_tools/decompile/parse_dashboard_kbn.py

```
@@ -0,0 +1,723 @@
+    state_dict: dict[str, Any] = state_model.model_dump(exclude_none=True, by_alias=True)
+    is_esql = raw_panel_type == 'esql' or _has_text_based_query(state_dict)
+    panel_type = 'esql' if is_esql else 'lens'
+
+    vis_state = _parse_visualization_state(attributes_raw, is_esql=is_esql)
+    form_layers = _parse_form_based_layers(state_dict) if not is_esql else {}
+    esql_layers = _parse_esql_layers(state_dict) if is_esql else {}
+    esql_query = _extract_esql_query_from_state(state_dict) if is_esql else None
+
+    refs_raw_from_attrs = get_list(attributes_raw, 'references') or []
+    refs_raw_from_config = get_list(embeddable_config_raw, 'references') or []
+    refs_raw = refs_raw_from_attrs if refs_raw_from_attrs else refs_raw_from_config
```

Ruff (0.15.6)

[error] Replace ternary `if` expression with `or` operator

Replace with `or` operator

(FURB110)